static/sass/config/_sections.scss

```diff
@@ -0,0 +1,43 @@
+// Section Config
+// ==============
+
+
+// Section Arrow
+// -------------
+/// Apply a top-border with centered down-arrow triangle,
+/// for use as a section-divider.
+/// @group sections
+/// @parameter $color -
```

Comment:
since we have a default, maybe we could include/list it in the @parameter definitions for both color and background?